### PR TITLE
Initialise aws mgn service in frankfurt region

### PR DESCRIPTION
**Pull request by Cloudyeli**
Migration VPC and Subnets are created. VPC Peering is in its own folder.

### CODE REVIEW

File: new_network/VPC_Peering/vpc_peering.tf [hunk 1]
```diff
@@ -0,0 +1,94 @@
+resource "aws_vpc_peering_connection" "migration" {
+  # peer_owner_id = var.peer_owner_id ### uses by default the owner ID of Owner Account
```
Comment (Jeena-J) [inline new_network/VPC_Peering/vpc_peering.tf:2]: remove commented code if not required.

**Review by Jeena-J — CHANGES_REQUESTED** [review]

**Comment by Jeena-J** [comment]
Minor changes. 
- Apply terraform fmt command on all files. 
- add the tf lock file to .gitignore file so that they dont get included in the commit.

Please check the inline comments as well.

**Comment by Jeena-J** [comment]
Reopening accidentally closed PR

**Review by Sadafpara — APPROVED** [review]
Looks good to me!

**Comment by Sadafpara** [comment]
There is an unresolved conflict

**Review by Jeena-J — APPROVED** [review]

**Pull request merged**